public/style/components/_card.scss

```diff
@@ -0,0 +1,14 @@
+.card {
```

Comment:
This is mostly a code finding thing, makes it easier to relate the final product back to the source. I was thinking that the `.container` class should site in it's own `_container.scss` file also